packages/letters-of-credit/src/components/LetterOfCredit/LetterOfCredit.js

```diff
@@ -51,8 +52,21 @@ class LetterOfCredit extends Component {
     .catch(error => {
       console.log(error);
     });
+    this.getLetter();
   }
 
+  getLetter() {
```

Comment:
I'm not sure why this function is necessary. The current LOC object is already stored in this.props.letter, so why do we need to make another REST call for the same letter?